pages/common/bash.md

```diff
@@ -28,6 +28,10 @@
 
 `bash -s`
 
+- Pass arguments behind to the script (can be combined with `-s` to pass arguments to commands/scripts from `stdin`):
```

Comment:
This seems a little vague IMHO, How about we start like:
`Stop processing options for bash....`

Reply:
Ya, this is better. I am going to change this.

Reply:
I've changed the description, is the new one better?